docs/dev/fundamentals/faq.md

```diff
@@ -157,6 +157,15 @@ At the moment, we support any Ethereum-based wallet. By default, the provided op
 To access the testnet funds ([Faucet](https://portal.zksync.io/faucet)), you can tweet about us and get tokens. Make sure that Twitter message contains your Ethereum address to which we’ll send funds and that it’s not your main Ethereum account. The faucet will not work with new Twitter accounts & accounts without avatar.
 
 Alternatively, you can use [our bridge](https://portal.zksync.io/bridge) to bridge ETH from Goerli to zkSync alpha testnet.
+
+## Why do i get wallet_requestPermissions error?
+
+This happens in when interacting with our [quickstart example](../developer-guides/hello-world.md#front-end-integration),
```

Comment:
There seems to be some english errors (or at least the overall writing can be improved) in these paragraphs.

Also, would add this as a last section in the quickstart tutorial instead the FAQ